packages/react-scripts/config/webpack.config.js

```diff
@@ -789,18 +795,6 @@ module.exports = function (webpackEnv) {
           },
         }),
     ].filter(Boolean),
-    // Some libraries import Node modules but don't use them in the browser.
-    // Tell webpack to provide empty mocks for them so importing them works.
-    node: {
```

Comment:
This is a breaking change - not sure if we are good by doing that in a minor version bump? (imho we should not confuse new developers by depending on node js builtin shims on the front-end - npm is confusing enough as is)